.github/workflows/bump-and-release.yml

```diff
@@ -0,0 +1,115 @@
+name: Bump and release
+run-name: ${{ (inputs.version == '' && 'Not bumping version') || format('Bumping version to {0}', inputs.version) }} and ${{ (inputs.dryRun == true && 'validating') || 'releasing' }} ${{ inputs.connectors }}
+on:
+  workflow_dispatch:
```

Comment:
If you replicate this block with "workflow_call" instead of "workflow_dispatch", we can also trigger the job manually from GithHub (if we want)